source/cleaning_service.rb

```diff
@@ -0,0 +1,82 @@
+# astia au mai multe statii de curatare
+# un orar in care functioneaza => hash cu zilele saptamanii
+# masinile programate
+require_relative 'time_table'
+
+class CleaningService
+  # attr_accessor :station_count 
+  # attr_accessor :car_count
+  attr_accessor :time_table
+  attr_accessor :stations
+  attr_accessor :queue
+
+  def initialize()
+    @stations = nil
+    @queue = nil
+    @time_table = TimeTable.new
+  end
+
+  def to_s
+    result = "Cleaning Service\n"
+    stations.each {|station| result += station.to_s}
```

Comment:
try to leave an empty space before and after the curly brackets

`stations.each { |station| result += station.to_s }`